### PR TITLE
Add council and democracy module

### PR DESCRIPTION
**Pull request by philipstanislaus**

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -302,11 +302,80 @@ impl pallet_staking::Trait for Runtime {
...
 }
 
+parameter_types! {
+	pub const LaunchPeriod: BlockNumber = 28 * 24 * 60 * MINUTES;
+	pub const VotingPeriod: BlockNumber = 28 * 24 * 60 * MINUTES;
+	pub const EmergencyVotingPeriod: BlockNumber = 3 * 24 * 60 * MINUTES;
+	pub const MinimumDeposit: Balance = 100 * 1_000_000_000_000_000_000;
+	pub const EnactmentPeriod: BlockNumber = 30 * 24 * 60 * MINUTES;
+	pub const CooloffPeriod: BlockNumber = 28 * 24 * 60 * MINUTES;
+	pub const PreimageByteDeposit: Balance = 1 * 10_000_000_000_000_000;
```
Comment (vedhavyas) [inline runtime/src/lib.rs]: do we have these parameters documented somewhere to look at?

File: src/chain_spec.rs [hunk 2]
```diff
@@ -134,20 +129,7 @@ impl Alternative {
                             get_authority_keys_from_seed("Bob"),
                         ],
                         get_account_id_from_seed::<sr25519::Public>("Alice"),
-                        vec![
```
Comment (vedhavyas) [inline src/chain_spec.rs]: Did we just moved them around or something changed here ?

File: runtime/src/lib.rs [hunk 3]
```diff
@@ -593,3 +665,57 @@ impl_runtime_apis! {
 		}
 	}
 }
+
+#[cfg(test)]
+mod tests {
```
Comment (vedhavyas) [inline runtime/src/lib.rs]: are these tests part of the Council and Democracy?

File: src/chain_spec.rs (hunk 2 above)
Comment (philipstanislaus) [reply src/chain_spec.rs]: Yes, we have a spreadsheet, which is pinned to our #crcl_flim on Slack.

File: runtime/src/lib.rs (hunk 1 above)
Comment (philipstanislaus) [reply runtime/src/lib.rs]: Yes, we have a spreadsheet, which is pinned to our #crcl_flim on Slack.

File: src/chain_spec.rs (hunk 2 above)
Comment (philipstanislaus) [reply src/chain_spec.rs]: These were moved here: https://github.com/centrifuge/centrifuge-chain/pull/144/files/3710754192fe895eb86de34533692a05e2511cec#diff-6740deafebf0ba9153f241785add2eefR240

File: runtime/src/lib.rs (hunk 3 above)
Comment (philipstanislaus) [reply runtime/src/lib.rs]: No, but they were to small for their own PR. I just copied them over from substrate.

**Comment by philipstanislaus** [comment]
@vedhavyas want to approve this?

**Pull request merged**